filebeat/config/config.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Defaults for config variables which are not set
 const (
-	DefaultRegistryFile                      = ".filebeat"
+	DefaultRegistryFile                      = "registry"
```

Comment:
> Those installations were using a non-default value so they should not be affected by the change.

Good point. But if it's just `registry` I can get rid of one of those nasty `sed` commands on the configuration file at package time by just setting the data path in the init script.